### PR TITLE
Display map based on number of projects per country

### PR DESCRIPTION
**Pull request by rickypchen**
@esoergel @czue 
DimagiSphere should display a heatmap with countries colored in based on the number of active projects

### CODE REVIEW

**Comment by esoergel** [comment]
looks good to merge on build.  I'm not crazy about the name `country_projects_count`, maybe `projects_per_country`? (not blocking)

File: corehq/apps/hqadmin/reports.py [hunk 1]
```diff
@@ -871,6 +873,18 @@ def format_date(dstr, default):
                     dom.get('internal', {}).get('sub_area') or _('No info')
                 ]
 
+    def _calc_num_projs_per_countries(self):
+        num_projects_by_country = (DomainES()
+                                   .terms_aggregation('deployment.countries', 'countries')
+                                   .run().aggregations.countries.counts_by_bucket())
```
Comment (czue) [inline corehq/apps/hqadmin/reports.py]: I think we should add `.size(0)` to the query to only return the aggregations

**Comment by NoahCarnahan** [comment]
Looks good, but has a merge conflict now

File: corehq/apps/hqadmin/static/hqadmin/js/project_map.js [hunk 2]
```diff
@@ -79,19 +88,25 @@ var projectMapInit = function(mapboxAccessToken) {
...
         that.toggleMap = function () {
             is_project_count_map = !is_project_count_map;
         };
 
-        that.getMaxNumProjects = function () {
-            return maxNumProjects;
-        };
+        that.getUnit = function (count) {
+            if (is_project_count_map) {
+                return count > 1 ? 'projects' : 'project'
```
Comment (dimagimon) [inline corehq/apps/hqadmin/static/hqadmin/js/project_map.js]: Missing semicolon. (semi)

File: corehq/apps/hqadmin/static/hqadmin/js/project_map.js [hunk 3]
```diff
@@ -79,19 +88,25 @@ var projectMapInit = function(mapboxAccessToken) {
...
+            } else {
+                return count > 1 ? 'users' : 'user'
```
Comment (dimagimon) [inline corehq/apps/hqadmin/static/hqadmin/js/project_map.js]: Missing semicolon. (semi)

File: corehq/apps/hqadmin/static/hqadmin/js/project_map.js [hunk 4]
```diff
@@ -79,19 +88,25 @@ var projectMapInit = function(mapboxAccessToken) {
...
+            }
+        }
```
Comment (dimagimon) [inline corehq/apps/hqadmin/static/hqadmin/js/project_map.js]: Missing semicolon. (semi)

**Pull request merged**